### ISSUE TITLE
MDNS

### ISSUE THREAD

**Issue by eliassal** [ntop/ntopng#4825]
Can you please tell me what does this alert means? Is it the winForDockerUpdate machine who is generating this flow? I understand that mdns is used for local broadcast domain in case of non-existance of DNS. 
clicking on any "_services._dns-sd._udp.l...   :5353", it displays details for a remote host through my router (2nd snapshot). Can you please help in translating this in plain english as I am completly confused with this protocol, maybe I am missing something regarding the protocol not ntopng maybe 

![mdnsagain](https://user-images.githubusercontent.com/8447864/101893296-d807d780-3ba4-11eb-8263-3cb41d0b94ab.jpg)

![ExternalMDNS](https://user-images.githubusercontent.com/8447864/101893890-ac392180-3ba5-11eb-8300-e9ddb9883dcd.jpg)

**Comment by lucaderi** [comment]
Please provide a pcap file wo we can reproduce it. Attach it to github please

**Comment by eliassal** [comment]
Hi Luca, hereis the 1st pcap file when I click on the name of the host winfordockerupdate (VM hosting ntopng), However, when I click on the _services._dns-sd._udp.l... , I get the following detals but downloading pcap get stuck at 24B with a circle continnously running and never finishes as you can notice in the snapshot, so I clicked on json and saved it to a json file attached here
![downloadStuck](https://user-images.githubusercontent.com/8447864/102064003-0d0b6880-3df7-11eb-990b-5b7df6036654.jpg)


[NPF_{1CF6F05B-8257-4458-84EC-4A0FD242FBD9}_192.168.1.19_live (3).zip](https://github.com/ntop/ntopng/files/5687798/NPF_.1CF6F05B-8257-4458-84EC-4A0FD242FBD9._192.168.1.19_live.3.zip)

The json file
[services._dns-sd_udp.zip](https://github.com/ntop/ntopng/files/5687809/services._dns-sd_udp.zip)

**Comment by lucaderi** [comment]
After analysis your problem is due to the fact you're contacting hosts that are blacklisted. For the 5353 issue I cannot say much as there not such packets in the pcap you attached

**Comment by eliassal** [comment]
OK thanks
